common/config/rush/pnpm-lock.yaml

```diff
@@ -10118,6 +10122,47 @@ packages:
       integrity: sha512-mjYaV8s2WtZjhutOHaUUXmi2Ne5i+O/doHq+xoMo8aqmB50wVLSdyG52mRA3JITqy667rK95QQdjyzG6LflpOw==
       tarball: file:projects/perf-storage-file-share.tgz
     version: 0.0.0
+  file:projects/purview-catalog.tgz:
+    dependencies:
+      '@azure/identity': 1.3.0
```

Comment:
Should the RLC clients obey the same standard as our typical track 2 packages? In which case azure/identity is a devDependency, and not a direct 'dependency'?

Reply:
Oh yeah that is a bit weird, it looks like pnpm-lock doesn't make a distinction between dev and non-dev dependencies. There is an issue in rush about this: https://github.com/microsoft/rushstack/issues/1516

There seems to be some interest specially for the audit scenario to be able to identify non-dev vulnerable dependencies repo wide.

Reply:
Oops, I should have gone directly to the source and not stared directly into the Matrix on that one. Resolving...